linkerd/config/src/main/scala/io/buoyant/linkerd/config/types/DtabDeserializer.scala

```diff
@@ -11,3 +11,11 @@ class DtabDeserializer extends ConfigDeserializer[Dtab] {
       Dtab.read(_parseString(parser, ctx))
     }
 }
+
+class DtabSerializer extends ConfigSerializer[Dtab] {
```

Comment:
These classes are so small that I find it more readable to put them in the same file.  I don't have a strong opinion about this, though.  Happy to change.

(Precedent: we violate the "one class per file" guideline all over the place)